### PR TITLE
DOC-4751 - burning tokens

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/tools/digital-currencies/burning/_index.md [hunk 1]
```diff
@@ -14,6 +14,67 @@ title: "Burning Tokens"
...
+	  alt="Burn Tokens - Flow Running"
+   >}}  
+   
+   Once the Burn Tokens flow completes, the message *Tokens have been successfully burned, check your vault balance for new balance* is displayed:
+
+   {{< 
+      figure
+	  src="images/tokens-have-been-successfully-burned-message.png"
+      width=50%
+	  figcaption="'Tokens have been successfully created...' Message"
```
Comment (AnnaK-R3) [inline content/en/tools/digital-currencies/burning/_index.md:68]:
```suggestion
	  figcaption="'Tokens have been successfully burned...' Message"
```

File: content/en/tools/digital-currencies/burning/_index.md [hunk 2]
```diff
@@ -14,6 +14,67 @@ title: "Burning Tokens"
 
 ## Burning Tokens
 
-The last stage of the token life cycle is *redemption*. An entity on the network can send a request to redeem their tokens for collateral at any time. During redemption, the tokens are 'burnt', or removed from circulation. 
+The introduction of {{< tooltip >}}CBDC{{< definitiondc term="CBDC" >}}{{< /tooltip >}} will inject a new source of liquidity into the economy. However, many central banks however peg their currency to the US dollar and as such hold dollar reserves to stabilize their currency. As a result, central banks will have a minimum threshold of foreign reserves - this can differ country to country.
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/tools/digital-currencies/burning/_index.md:17]: Done

File: content/en/tools/digital-currencies/burning/_index.md [hunk 3]
```diff
@@ -14,6 +14,67 @@ title: "Burning Tokens"
...
+   >}}  
+   
+   Once the Burn Tokens flow completes, the message *Tokens have been successfully burned, check your vault balance for new balance* is displayed:
+
+   {{< 
+      figure
+	  src="images/tokens-have-been-successfully-burned-message.png"
+      width=50%
+	  figcaption="'Tokens have been successfully created...' Message"
+	  alt="'Tokens have been successfully created...' Message"
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/tools/digital-currencies/burning/_index.md:69]: Done

File: content/en/tools/digital-currencies/burning/_index.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/tools/digital-currencies/burning/_index.md:68]: Done

File: content/en/tools/digital-currencies/burning/_index.md [hunk 4]
```diff
@@ -14,6 +14,67 @@ title: "Burning Tokens"
...
+	  alt="Token Manager Page"
+   >}}
+   
+   The page contains the following panes:
+   
+   * **Vault Balances:** Displays the number of tokens available for each token definition for the current participant.
+   * **Mint Tokens:** Enables you to mint tokens; see [Minting Tokens]({{<relref "../minting/_index.md" >}}).
+   * **Burn Tokens:** Enables you to burn tokens.
+      
+4. In the **Burn Tokens** pane, specify the following values:
```
Comment (yarivus) [inline content/en/tools/digital-currencies/burning/_index.md:41]: where is (2) and (3) number items ?

Comment (paulmoloneyr3) [reply content/en/tools/digital-currencies/burning/_index.md]: Cheers - fixed numbering.

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**